src/js/jsx/sections/transform/Rotate.jsx

```diff
@@ -50,18 +52,36 @@ define(function (require, exports, module) {
          */
         _lastAngle: null,
```

Comment:
I see the comment at line 168, but that doesn't seem to explain why we can't just use state.value